components/webui/imports/api/ingestion/collections.js

```diff
@@ -12,7 +12,12 @@ const STATS_COLLECTION_ID = Object.freeze({
 
 const StatsCollection = new Mongo.Collection(Meteor.settings.public.StatsCollectionName);
 
+const CompressionJobsCollection = new Mongo.Collection(
```

Comment:
Could move this above the stats related variables, just for ordering consistency.